test/utilsTest.js

```diff
@@ -34,13 +35,13 @@ describe("utils", function() {
 		it("throws on undefined origin", function() {
 			assert.throws(() => {
 				checkOrigin(undefined);
-			}, Error, "Invalid URL: undefined");
+			}, Error, "Invalid URL");
```

Comment:
Why remove the specification of which URL it was? This would be useful for debugging, isnt it?

Reply:
this was causing build failures in node v14 since there native URL class doesn't include invalid url on error message, I've adjusted it to be compatible with node v14 and above, should I restore it?

tldr:
error message thrown on node v14: `Invalid URL`
error message thrown on node v15: `Invalid URL: asdf`

Reply:
Didn't know that this was the behavior in node v14 - just to be clear, we are not talking about the versions above, right? But it doesn't matter that much, as I was mostly worrying about debugging information, but missed that it still has its own test case.